lib/rules/function-calc-no-unspaced-operator/index.js

```diff
@@ -48,6 +48,8 @@ const rule = (primary, _secondaryOptions, context) => {
 		}
 
 		root.walkDecls((decl) => {
+			if (!OPERATOR_REGEX.test(getDeclarationValue(decl))) return;
+
 			let needsFix = false;
 			const valueIndex = declarationValueIndex(decl);
 			const parsedValue = valueParser(getDeclarationValue(decl));
```

Comment:
[suggest] How about reusing a return value from `getDeclarationValue(decl)` like this?

```suggestion
			const value = getDeclarationValue(decl);

			if (!OPERATOR_REGEX.test(value)) return;

			let needsFix = false;
			const valueIndex = declarationValueIndex(decl);
			const parsedValue = valueParser(value);
```

Reply:
good catch!